xxhash.h

```diff
@@ -2149,12 +2158,14 @@ typedef XXH64_hash_t xxh_u64;
  * also slightly faster because it fits into cache better and is more likely
  * to be inlined by the compiler.
  *
+ * Unrolling XXH64 is also disabled on AArch64. While it is a 64-bit platform,
```

Comment:
Q: Is there a scenario (architecture) in which unrolling XXH64 finalization is a net gain ?

Reply:
I'm starting to think that this isn't the case.

Was the unrolling added before XXH64?

Reply:
The unrolling was added after `XXH64` creation, but before `XXH3` existed.

I think the contributor was mostly interested in `XXH32`, and stated that it was beneficial for speed on his target.
The `XXH64` extension was more of stretch, along the line "if it works for `XXH32`, it should work for `XXH64`".

Thing is, back in these days, my ability to just measure such an impact was quite limited : xxHash was still very much intended as a checksum algorithm for large input. So I could not really tell if it was better or worse on small data, it just seemed "about the same" (I also did not pay attention to the binary size increase).

Anyway, today I've got a benchmark program specifically created for comparison small data, so let's use that and compare.